### PR TITLE
 Completed

### PR DESCRIPTION
**Pull request by Alex-Davis0**
Screenshot 2022-01-31 163847
Screenshot 2022-01-31 164310
Screenshot 2022-01-31 163149

### CODE REVIEW

File: es6-classes/example-class.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class ExampleClass {
+  constructor() {
+    this.name = name;
+  }
+}
```
Comment (scbowler) [inline es6-classes/example-class.js]: This should be empty, we don't want to arbitrary add things.

```JAVASCRIPT
class ExampleClass {}
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good! I small update needed, let me know if you have any questions.

**Comment by Alex-Davis0** [comment]
Screenshot 2022-01-31 164310

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**